src/shared_modules/keystore/src/keyStore.cpp

```diff
@@ -40,6 +40,38 @@ void Keystore::put(const std::string& columnFamily, const std::string& key, cons
     }
 }
 
-void Keystore::get(const std::string& columnFamily, const std::string& key, rocksdb::PinnableSlice& value)
+void Keystore::get(const std::string& columnFamily, const std::string& key, std::string& value)
 {
+    std::string encryptedValue;
+
+    try
+    {        
+        // Get from DB
+        Utils::RocksDBWrapper keystoreDB = Utils::RocksDBWrapper(DATABASE_PATH, false);
+
+        if(!keystoreDB.columnExists(columnFamily)) {
+            char msg[100];
+            sprintf(msg, "Column %d does not exists in the database.", columnFamily.c_str());
+            logError(KS_NAME, msg);
+            throw std::runtime_error(msg);
+        }
+
+        if (!keystoreDB.get(key, encryptedValue, columnFamily)) {
+            char msg[100];
+            sprintf(msg, "Could not find key '%s' in column '%s'.", key, columnFamily);
```

Comment:
NIT: I think is nittier `Could not find key '%s' at column '%s'.` instead of `Could not find key '%s' in column '%s'.`

Reply:
What would happen if key and olumnFamily together sums more than 70 bytes? I think asprintf would be a better option, of snprintf & some dynamic allocation method if not available in all environments